actstream/managers.py

```diff
@@ -190,14 +190,16 @@ def following_qs(self, user, *models, flag=''):
             ctype_filters |= Q(content_type=ContentType.objects.get_for_model(model))
         qs = qs.filter(ctype_filters)
 
+        flag = kwargs.get('flag', '')
+
         if flag:
             qs = qs.filter(flag=flag)
         return qs.fetch_generic_relations('follow_object')
 
-    def following(self, user, *models, flag=''):
+    def following(self, user, *models, **kwargs):
```

Comment:
Which python version raises syntax error? 3.5 or 3.6?

Reply:
3.5

Reply:
I think even 2.7

Reply:
its not actually python version issue

Reply:
So what's the problem, I test my fork version in local py3.5 environment and get no errors. The modified codes work same as previous, I think, but a little more verbose.

Reply:
I thought keyword-only arguments were a syntax error in python 2.7 @auvipy?

```
  File "/home/travis/build/justquick/django-activity-stream/actstream/managers.py", line 180
    def following_qs(self, user, *models, flag=''):
                                             ^
SyntaxError: invalid syntax
```